src/main/java/org/apache/maven/plugins/jar/AbstractJarMojo.java

```diff
@@ -222,6 +253,7 @@ public File createArchive() throws MojoExecutionException {
         jarContentFileSet.setDirectory(getClassesDirectory().getAbsolutePath());
         jarContentFileSet.setIncludes(Arrays.asList(getIncludes()));
         jarContentFileSet.setExcludes(Arrays.asList(getExcludes()));
+        jarContentFileSet.setUseDefaultExcludes(addDefaultExcludes);
```

Comment:
As `jarContentFileSet` is only used for searching for multi release and module descriptor I will not add this line at all